base/doc/ltnews41.tex

```diff
@@ -507,100 +620,131 @@ \subsection{Fixing the spacing after display math}
 conditionals, such as \cs{IfSocketExistsTF}, to support such
 scenarios. Corresponding L3 programming layer conditionals are also
 provided.
+%% List them all, or give a reference! 
 %
 \githubissue{1577}
 
 
 \subsection{Accessing the current counter}
 
-Counter commands such as \cs{alph}, \cs{stepcounter}, may now have the
-argument \texttt{*} to denote the current counter (as used by
-\cs{label}).  This is compatible with the package \pkg{enumitem} use
-of \verb|\alph*| in item labels but is generally available.  Not all
-commands accept \verb|*|, for example \verb|\counterwithin| and
-\verb|\counterwithout| require counter names as before.
+Counter commands such as \cs{alph}, \cs{stepcounter}, can now use the
+argument \texttt{*} to denote the \emph{current counter} (in the sense used by
+\cs{label}).\marginpar{not sure: David?}
+This is compatible with the use by the \pkg{enumitem} package 
+of \verb|\alph*| in item labels; and it is now generally available.  Not all
+commands accept \verb|*|: for example, \verb|\counterwithin| and
+\verb|\counterwithout| still require counter names as before.
 %
 \githubissue{1632}
 
 \subsection{Collecting environment bodies verbatim}
 
-The mechanisms in \pkg{ltcmd} (\enquote{\pkg{xparse}}) offer a
+The mechanisms provided with \cs{NewDocumentCommand}, etc.\ (formerly \enquote{\pkg{xparse}}) offer a
 powerful way to specify a range of types of document command and
 environment syntax. This includes the ability to collect the entire
 body of an environment, for cases where treating it as a standard
-argument is useful. It is also possible in \pkg{ltcmd} to define
-arguments which grab their content verbatim, another specialist
-argument form. To date, however, it was not possible to combine these
+argument is useful. It is also possible using the mechanism
+to define
+arguments which grab their content verbatim.
+To date, however, it was not possible to combine these
 two ideas.
 
-In this release, a new specifier~\texttt{c} is introduced, which
-collects the body of an environment in a verbatim-like way. Like the
+In this release a new specifier,~\texttt{c},
+for use
+in \cs{NewDocumentEnvironment} and friends
+has been introduced: 
+%% used where?
+this collects the body of an environment in a verbatim-like way. 
+As for the
 existing \texttt{+v}~specification, each separate line is marked by
 the special \cs{obeyedline} marker, which as standard issues a normal
 paragraph. Thus, this new specifier is usable both for typesetting and
-collecting file contents (the letter~\texttt{c} indicates
-\enquote{collect code}). Thus, we may use
+for collecting file contents (the letter~\texttt{c} indicates
+\enquote{collect code}). 
+%% the example below is very bad as an example of typesetting !! 
+%%  and its content is even worse! In fact it is "nonsense"!! 
+%% Why the ... is it centred ???
+%%FMi:  center is indeed problematical as it looks like c - tried flushright
+\marginpar{Joseph?}
```

Comment:
I was simply picking an example of something relatively awkward to pull off without `c` that is easy with!